examples/repl/stress.rs

```diff
@@ -442,50 +444,25 @@ pub unsafe extern "C" fn stress_functions(mut context: &dc_context_t) {
             );
         } else {
         };
+
         let mut buf: *mut libc::c_void = 0 as *mut libc::c_void;
         let mut buf_bytes: size_t = 0;
-        if 0 != (0
-            == dc_read_file(
+
+        assert_eq!(
+            dc_read_file(
                 context,
                 b"$BLOBDIR/dada\x00" as *const u8 as *const libc::c_char,
                 &mut buf,
                 &mut buf_bytes,
-            )) as libc::c_int as libc::c_long
-        {
-            __assert_rtn(
-                (*::std::mem::transmute::<&[u8; 17], &[libc::c_char; 17]>(b"stress_functions\x00"))
-                    .as_ptr(),
-                b"../cmdline/stress.c\x00" as *const u8 as *const libc::c_char,
-                253i32,
-                b"dc_read_file(context, \"$BLOBDIR/dada\", &buf, &buf_bytes)\x00" as *const u8
-                    as *const libc::c_char,
-            );
-        } else {
-        };
-        if 0 != !(buf_bytes == 7) as libc::c_int as libc::c_long {
-            __assert_rtn(
-                (*::std::mem::transmute::<&[u8; 17], &[libc::c_char; 17]>(b"stress_functions\x00"))
-                    .as_ptr(),
-                b"../cmdline/stress.c\x00" as *const u8 as *const libc::c_char,
-                254i32,
-                b"buf_bytes==7\x00" as *const u8 as *const libc::c_char,
-            );
-        } else {
-        };
-        if 0 != !(strcmp(
-            buf as *const libc::c_char,
-            b"content\x00" as *const u8 as *const libc::c_char,
-        ) == 0i32) as libc::c_int as libc::c_long
-        {
-            __assert_rtn(
-                (*::std::mem::transmute::<&[u8; 17], &[libc::c_char; 17]>(b"stress_functions\x00"))
-                    .as_ptr(),
-                b"../cmdline/stress.c\x00" as *const u8 as *const libc::c_char,
-                255i32,
-                b"strcmp(buf, \"content\")==0\x00" as *const u8 as *const libc::c_char,
-            );
-        } else {
-        };
+            ),
+            1
+        );
+        assert_eq!(buf_bytes, 7);
```

Comment:
Very nice! I was wondering about these odd looking asserts and just now realized they are basically saying which file and line it happened.

Reply:
yeah, we really need to convert these to make this file more sane